lib/components/UserProfile/index.jsx

```diff
@@ -17,6 +16,22 @@ export class UserProfile extends Component {
     this.props.fetchUserData(id, token)
   }
 
+  stopSound = () => {
+    this.props.stopSound()
+  }
+
+  loadSounds(){
+   return this.props.userData.sounds.map((sound, i) => {
+     let spec = JSON.parse(sound.attributes)
+     let title = spec.soundName ? spec.soundName : 'untitled'
+     return <div>
+              <h3 key={i} id={sound.id}>Sound: {title}</h3>
+                <button onClick={(e) => this.props.previewSound(spec)}>Play</button>
+                <button onClick={this.stopSound}>Stop</button>
```

Comment:
Should we toggle the button to show "Play" or "Stop" depending on a boolean in state? Made a change do that with "Play" / "Pause"  on the sequencer and it's pretty clean. Otherwise this seems straight forward!